.github/workflows/ci.yml

```diff
@@ -98,6 +125,20 @@ jobs:
           echo "::set-env name=CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER::/usr/bin/aarch64-linux-gnu-gcc-5"
           echo "::set-env name=QEMU_LD_PREFIX::/usr/aarch64-linux-gnu"
 
+      - name: Install rust-embedded/cross
+        # see https://github.com/rust-embedded/cross
+        if: contains(matrix.config.target, 'android')
+        run: |
+          cargo install cross
+          echo 'alias cargo=cross' >> $HOME/.bash_profile
```

Comment:
Does this work?
If so, is it still necesary to use `{{ matrix.config.cargo }}` everywhere?

Reply:
Nope, I forgot to remove that line, it didn't work

Reply:
Maybe I missed something, I will recheck that